Microsoft.DotNet.Interactive.CSharp/CSharpKernel.cs

```diff
@@ -8,6 +8,7 @@
 using System.Reflection;
 using System.Text;
 using System.Threading.Tasks;
+using Interactive.DependencyManager;
```

Comment:
The full namespace should probably be `Microsoft.DotNet.Interactive.DependencyManager`.

Reply:
This is an FSharp component.  There we don't name anything Microsoft.

Reply:
We can go back to FSharp, if you want or I suppose we can use Microsoft in this case if that is important, there is actually nothing specifically stopping us.

Reply:
Assuming this is considered a Microsoft component I would probably go with Microsoft. FWIW, it seems this component is meant to have extension points for a variety of languages. For our own sanity I think it would makes to use uniform naming for public APIs that the components are expected to use.